builtin/providers/aws/resource_aws_eip.go

```diff
@@ -102,8 +102,8 @@ func resourceAwsEipRead(d *schema.ResourceData, meta interface{}) error {
 	domain := resourceAwsEipDomain(d)
 	id := d.Id()
 
-	assocIds := []*string{}
-	publicIps := []*string{}
+	var publicIps []*string
+	var assocIds []*string
```

Comment:
The test was failing to create the EIP because of this. While this fixes this particular acceptance test, #1598 may be a better solution long term